common/lib/xmodule/xmodule/modulestore/tests/test_cross_modulestore_import_export.py

```diff
@@ -345,6 +345,7 @@ def asset_collection(self):
     'manual-testing-complete',
     'split_test_module',
     'split_test_module_draft',
+    'split_test_module_static_tabs',
```

Comment:
Also this test doesn't fail without the patch. The problem is that this test compares a course imported from XML and then that course when it's been exported/imported.

We need a test that creates a course in split, exports that course, then reimports it, and checks that the url_name changes from whatever its run was to `course`.